src/Sylius/Bundle/ApiBundle/DataTransformer/LoggedInCustomerEmailAwareCommandDataTransformer.php

```diff
@@ -42,22 +44,25 @@ public function transform($object, string $to, array $context = [])
         return $object;
     }
 
+    /**
+     * @param mixed $object
+     */
     public function supportsTransformation($object): bool
     {
         return $object instanceof CustomerEmailAwareInterface;
     }
```

Comment:
Will it work?

```suggestion
    public function supportsTransformation(object $object): bool
    {
        return $object instanceof CustomerEmailAwareInterface;
    }
```

Otherwise it will crash if it's not a object, so either type should be `object`, not `mixed` or check should test for `$object` type as well.